packages/governance/src/committee.js

```diff
@@ -26,6 +27,11 @@ const { ceilDivide } = natSafeMath;
  * }} CommitteeElectorateCreatorFacet
  */
 
+export const privateArgsShape = {
```

Comment:
This is unused. Don't we need a `mustMatch()` in `start()`?

Reply:
https://github.com/Agoric/agoric-sdk/commit/be31fbc16d913878780af0ecf5fe526f88994f7b

Reply:
I forgot they were extracted from `evaluateContract()`.  That's going to be hard to remember.

Would it make sense to include a type declaration to show the reader that it's not misspelled, and to give a place to put javadoc saying how it gets used?
```
  /** @type {PrivateArgsShape} */
  ...
  /** @type {CustomTermsShape} */
```

Reply:
perhaps. seems akin to the `export const start` that it's just something to know about contract modules.

Reply:
Oh, I saw that note. I just thought that it wasn't passed anywhere. A type declaration with a comment explaining the magic would have resolved that, I think.

> seems akin to the `export const start` that it's just something to know about contract modules.
yeah, but `start` is the same in every contract. Maybe when most contracts use these magic shape names I won't need a reminder.